src/components/presentational/ValueSelector/ValueSelector.css

```diff
@@ -0,0 +1,76 @@
+.mdhui-value-selector {
+    --mdhui-value-selector-title-color: var(--mdhui-text-color-1);
+    --mdhui-value-selector-subtitle-color: var(--mdhui-text-color-3);
+    --mdhui-value-selector-value-background-color: #fff;
+    --mdhui-value-selector-value-text-color: var(--mdhui-text-color-1);
+    --mdhui-value-selector-button-value-selected-background-color: var(--mdhui-color-primary);
+    --mdhui-value-selector-button-value-selected-text-color: #fff;
```

Comment:
This will break it in dark mode, since the other mdhui colors adjust automatically, so this will end up as white on white

Reply:
Overall I would just test and make sure the default colors work nicely in dark mode...understand that all the colors ColorDefinition would let you configure it look nice in dark mode but I think it should look good without any of that

Reply:
so just here for the #FFF values I would probably use the CSS var for that as well

Reply:
I modified `--mdhui-value-selector-value-background-color` to use a variable, but I left `--mdhui-value-selector-button-value-selected-text-color` as `#fff`.  I don't see a variable for white text at the moment.  I could add one, but in this case, I'd actually want it to remain white since the primary color does not adjust for light vs dark mode.

<img src="https://github.com/CareEvolution/MyDataHelpsUI/assets/5408603/0aeafa63-9425-4885-8e25-3e218e91fed4" width="350px" /><img src="https://github.com/CareEvolution/MyDataHelpsUI/assets/5408603/83d92049-3c1a-48a1-8b26-4391f646c9b5" width="350px" />

Reply:
Right, yes...same thing for normal mdhui-buttons